src/plays/expanding-cards/ExpandingCards.css

```diff
@@ -0,0 +1,75 @@
+.play-container {
+  width: 100%;
+  margin: 0 auto;
+  text-align: center;
+}
+.card-container {
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+}
+
+.Expanding-card {
+  width: 80px;
+  height: 400px;
+  cursor: pointer;
+  margin: 20px;
+  transition: width 1s;
+  position: relative;
+}
+
+.active {
+  width: 600px;
+}
+img {
```

Comment:
Do not override the selector directly. Scope it with a class name relevant to your play.